src/aws-provider/aws-organization-reader.ts

```diff
@@ -422,6 +499,7 @@ export class AwsOrganizationReader {
 
     public readonly policies: Lazy<AWSPolicy[]>;
     public readonly accounts: Lazy<AWSAccount[]>;
+    public readonly partitionAccounts: Lazy<AWSAccount[]>;
```

Comment:
if the reader runs in the context of govcloud, would it not be possible to:
1) store the accounts in accounts (instead of partitionAccounts)
2) use Lazy.getValue (instead of Lazy.getPartitionValue)

I would expect we either run on a partition (e.g. govcloud) or we run on commercial - not both

Reply:
further down I realized (or it seems) that the AwsOrganization and OrganizationReader/Writer do get re-used. 

if this indeed is the case, it might be worth a try to instantiate two toplevel organizations, each with their own AwsOrganization instance, OrganizationReader and OrganizationWriter. this is minimise the number of code that needs to change within the classes to support "two of everything".